packages/powersync/test/schema_test.dart

```diff
@@ -106,36 +109,39 @@ void main() {
           greaterThan(versionAfter2['schema_version']));
     });
 
-    test('Indexing', () async {
-      final powersync = await setupPowerSync(path: path, schema: schema);
-
-      final results = await powersync.execute(
-          'EXPLAIN QUERY PLAN SELECT * FROM assets WHERE make = ?', ['test']);
-
-      expect(results[0]['detail'],
-          contains('USING INDEX ps_data__assets__makemodel'));
-
-      // Now drop the index
-      final schema2 = Schema([
-        Table('assets', [
-          Column.text('created_at'),
-          Column.text('make'),
-          Column.text('model'),
-          Column.text('serial_number'),
-          Column.integer('quantity'),
-          Column.text('user_id'),
-          Column.real('weight'),
-          Column.text('description'),
-        ], indexes: []),
-      ]);
-      await powersync.updateSchema(schema2);
-
-      // Execute instead of getAll so that we don't get a cached query plan
-      // from a different connection
-      final results2 = await powersync.execute(
-          'EXPLAIN QUERY PLAN SELECT * FROM assets WHERE make = ?', ['test']);
-
-      expect(results2[0]['detail'], contains('SCAN'));
-    });
+    /// TODO this test breaks on web at the moment.
```

Comment:
We should figure out why this is breaking. While this specific scenario may not be common in real apps, there could be an underlying issue that causes more weird bugs.

Reply:
Thanks for the investigation! This will be fixed in a future alpha release of `sqlite_async.dart`

Reply:
This test now passes after updates to the Drift worker